### DISCUSSION TITLE
Does the Apple provider work at all?

### DISCUSSION THREAD

**Discussion by Artur-Galstyan** [nextauthjs/next-auth#9090] [Help]
Has anyone gotten the Apple provider to work at all and if so, how? I'm getting [this issue](https://github.com/nextauthjs/next-auth/issues/6788) and the suggestion below in that issue doesn't work. So, can I assume that it simply doesn't work?

**Comment by misterioss** [comment]
Yes, it works. I used this manual: https://www.youtube.com/watch?v=Deyt6dJAjbE

**Comment by leo3linbeck** [comment]
It does not work if you redirect sign in flow (which I'm using for deployment on Vercel). See here: https://github.com/nextauthjs/next-auth/discussions/9942. Cheers, L3
